### PR TITLE
[OFE-21/22/23] Created the Salary Tracker page and appended an id in jobs for smooth scroll

### PR DESCRIPTION
**Pull request by KevJ2021**
To access the Salary Tracker page, navigate to 127.0.0.1:4000/salary-tracker after starting the local server.

### CODE REVIEW

File: _includes/salary-tracker.html [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+
+    // Main launch function
+    // Define the doc url and execute fetch promise
+    // Parse the returned data into a formatted/friendly array
+    // Remove the 'no positions loaded' selection option for positions
+    // Update the data
+    function main() {
+
+        var url = "https://docs.google.com/spreadsheets/d/e/2PACX-1vSr_TpN5LWs9ejhbmttNHfqG25LLi9rPKHPE3wYvovXwNNXSxcFsDSEBSEsEf-9MNfwFTKjLPh1FTLL/pub?output=csv";
+        var processedData = [];
```
Comment (richmungla) [inline _includes/salary-tracker.html:258]: Should url be a configurable property?

Comment (KevJ2021) [reply _includes/salary-tracker.html:258]: We should not really need to change the url as the sheet shouldn't change. And I'd prefer not to leave it change-able from within Site Leaf.

What are your thoughts as to why it should be configurable? Perhaps I may be missing a use-case

**Review by richmungla — APPROVED** [review]
LGTM

File: _includes/salary-tracker.html (hunk 1 above)
Comment (richmungla) [reply _includes/salary-tracker.html:258]: I had initially thought It is something that may change, since you have clarified that. It looks good to me 👍🏾

**Pull request merged**